e2eAIOK/common/trainer/train.py

```diff
@@ -4,6 +4,7 @@
 import torch
```

Comment:
Shall we put the trainer in the common folder, or define the separate trainer for DENAS and Model Adaptor?

Reply:
the trainer in common folder is the base trainer class, for training scripts, DENAS and Mode Adaptor can define their own training scripts. Maybe it is better to put train.py in DENAS/Model Adaptor folder

Reply:
Yes, I think putting the train.py in DENAS/Model Adaptor folder sounds good.

Reply:
move train.py to DENAS/Model Adaptor folder